Add config (#26)
yiisoft/translator-message-gettext: README.md

composer require yiisoft/translator-message-gettext
```

## Configuration

In case you use [`yiisoft/config`](http://github.com/yiisoft/config), you will get configuration automatically. If not, the following DI container configuration is necessary:

```php
<?php

declare(strict_types=1);

use Yiisoft\Translator\MessageReaderInterface;
use Yiisoft\Translator\Message\Gettext\MessageSource;
use Yiisoft\Aliases\Aliases;

return [
    MessageReaderInterface::class => [
        'class' => MessageSource::class,
        '__construct()' => [
            fn (Aliases $aliases) => $aliases->get('@message'),
        ],
    ],
];
```

> **Note:**  You can use absolutely path to translation files, if you not use [`yiisoft/aliases`](https://github.com/yiisoft/aliases)

Reviewer: ```suggestion
**Note:**  You can use absolute path to translation files in case you don't want to use [`yiisoft/aliases`](https://github.com/yiisoft/aliases)
```

```php
    MessageReaderInterface::class => [
        'class' => MessageSource::class,
        '__construct()' => [
            '/var/www/app/resourse/messages',
        ],
    ],
```

### Attention

> On using gettext, you need configure `locale` for package yiisoft/translator according to your OS requirement:

Reviewer: ```suggestion
When using gettext, `locale` depends on your OS requirement:
```
